### DISCUSSION TITLE
npm ERR! code 128

### DISCUSSION THREAD

**Discussion by Mj6078** [community/community#109470] [npm] [answered]
### Body

Hello to all, I am trying to access the stencil cli Theme file and try to install all dependencies but few dependencies I am unable to install and it is throwing the error npm ERR! code 128
npm ERR! An unknown git error occurred
npm ERR! command git --no-replace-objects clone -b v3.5.0 ssh://git@github.com/bigcommerce-labs/node-sass.git C:\Users\My PC\AppData\Local\npm-cache\_cacache\tmp\git-cloneXAk7fj --recurse-submodules --depth=1 --config core.longpaths=true
npm ERR! fatal: destination path 'C:\Users\My PC\AppData\Local\npm-cache\_cacache\tmp\git-cloneXAk7fj' already exists and is not an empty directory.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\My PC\AppData\Local\npm-cache\_logs\2024-02-21T15_03_05_151Z-debug-0.log. 
this for all other stencil theme environments is working properly but for peak theme, it is giving the same error Do you guys have any idea how to solve this?  Please help me on this error it is taking much of time to solve this. 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by its21yash** [comment]
Deleting and letting npm re-create the file package-lock.json resolved the issue.

`rm package-lock.json`

Clear npm cache: Sometimes, npm cache can cause unexpected issues. You can try clearing the npm cache by running the following command:
`npm cache clean --force`

A recommended step is to use the latest npm:
`npm install -g npm`

**Reply by Mj6078** [reply]
I Have already tried this But the problem is Still the Same If You have another approach to try please let me know. Thank You For your Answer.

**Reply by JothikaSampath** [reply]
Hi, Even I'm facing same issue, Did you resolve it

**Reply by Mj6078** [reply]
Hello, I found that I was installing the wrong dependencies node-saas was converted the name into only saas that's why it was giving an error but after installing saas now it is working fine.

**Reply by mandeep848** [reply]
> Hello, I found that I was installing the wrong dependencies node-saas was converted the name into only saas that's why it was giving an error but after installing saas now it is working fine.

so what coomand did you write for installing that dependency?

**Reply by Mj6078** [reply, marked as answer]
npm i sass  try this to install and then after try again.

**Comment by netshdev** [comment]
Try `npm config set legacy-peer-deps true` and then the `npm cache clean -f` and `npm install`.
